src/containers/offer-cards/OfferCards.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from '@mui/material'
+import { FC } from 'react'
+import OfferDetails from '~/pages/offer-details/OfferDetails'
+import { styles } from './OfferCards.style'
+
+const OfferCards: FC<{ isSquare: boolean }> = ({ isSquare = true }) => {
+  return (
+    <Box sx={isSquare ? styles.blockCard : styles.inlineCard}>
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
+      <OfferDetails isSquare={isSquare} />
```

Comment:
What are these 9 same lines of code

Reply:
it would be better to create and use a function with a name like dummyItems, that returns this jsx using loop. This way you'll state the purpose of the code, and also, the code will be cleaner because it won't contain repetitions.
And if you don't update the code right away, it will still be ok.

Reply:
Ok, I already done it's in another task. So I just merge and fix later.